Formula/soapysdr.rb

```diff
@@ -19,12 +19,15 @@ class Soapysdr < Formula
 
   depends_on "cmake" => :build
   depends_on "swig" => :build
-  depends_on "python@3.10"
+  depends_on "python@3.11"
+
+  def python3
+    "python3.11"
+  end
 
   def install
     args = %W[
-      -DENABLE_PYTHON=OFF
-      -DENABLE_PYTHON3=ON
+      -DPYTHON_EXECUTABLE=#{which(python3)}
```

Comment:
`ENABLE_PYTHON=OFF` + `ENABLE_PYTHON3=ON` disables all Python bindings since upstream only uses `swig/python3` if `ENABLE_PYTHON=ON` and it builds with python2. https://github.com/pothosware/SoapySDR/tree/master/swig/python3#build-python-version-3-bindings-only
> This directory is only activated when the top level python/ directory is configured to build for a Python version 2 environment.

---

The actual way to only enable Python 3-bindings is `ENABLE_PYTHON=ON`, `ENABLE_PYTHON3=OFF`, and CMake detecting corresponding Python 3 while configuring `swig/python` directory.

The `ENABLE_*` should automatically be set by CMake.

CMake does actually detect Python 3 without `PYTHON_EXECUTABLE` since we have added `opt_libexec/"bin"` to build PATH; however, I just added `PYTHON_EXECUTABLE` to be explicit.